src/components/TF_FIGURE/index.tsx

```diff
@@ -6,13 +6,15 @@ import TF_LONG_DESCRIPTION from "../tf_description";
 import { GraphData } from "../main_component";
 import Chart from "./Chart";
 import { level_boundary } from "./tr_utils";
+import { useQueryParam } from "use-query-params";
+import { mainQueryParamsConfig } from "../../app_config";
 
 export interface Props {
   colspan?: number;
   data: GraphData;
   figure_class?: string;
   selectedTreatmentUnits: string[];
-  update_selected_row(row: string): void;
+  update_selected_row(row: string | undefined): void;
```

Comment:
Do we have to change this to take undefined? Seems like nothing has changed in regards to this?

Reply:
undefined is (in useQueryParam) used to remove the param from the url. Don't think we need it now that I set a default and also validate. Type error is also gone now so I have removed it.